Configuration/TCA/Overrides/101_sys_template.php

```diff
@@ -29,16 +29,6 @@
     'Bootstrap Package: Content Elements'
 );
 
-/***************
- * TypoScript: Framework
- * Include Bootstrap 3.x (LESS) Assets
- */
-\TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addStaticFile(
-    'bootstrap_package',
-    'Configuration/TypoScript/Bootstrap3',
-    'Bootstrap Package: Bootstrap 3.x (LESS)'
-);
-
 /***************
```

Comment:
Should we keep the BP4 include? Not sure about BP5

Reply:
You're right, seems not to be useful currently

Reply:
Lets keep it, BP5 is almost here.